packages/website/docs/getting-started.md

```diff
@@ -58,15 +39,21 @@ If you do not wish to use a package manager, you can use standalone endpoints:
 <script src="https://unpkg.com/@algolia/autocomplete-js@alpha"></script>
```

Comment:
In our docs [we recommend using jsDelivr](https://www.algolia.com/doc/api-client/getting-started/update-the-client/javascript/?client=javascript#other-cdns), <e may want to remove this unpkg mention then.

Reply:
Good reference. I changed it to look more like our IS recommendations. https://github.com/algolia/autocomplete.js/pull/415/commits/97d23b3c863210f2d251a98b1e5bebe2cf14fe1d